android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -95,26 +99,39 @@ private static void doLaunch(Context context, HashMap<String, Object> extraData)
 
 
     public static void launch(Context context, HashMap<String, Object> extraData) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            PermissionListener listener = new PermissionListener() {
-                @Override
-                public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
-                    if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
-                        doLaunch(context, extraData);
-                    }
-
-                    return true;
+        List<String> permissionsList = new ArrayList<>();
+
+        PermissionListener listener = new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
+                if (results.length > 0 && results[i] == PackageManager.PERMISSION_GRANTED) {
```

Comment:
This needs to loop through all the results.

It seems like we're only checking the i-th result.

Does this get called multiple times?

Since we get a results array we need to check all of them once they are all resolved and only then launch.